test_automation/work_order/test_work_order_invalid_parameter.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+import logging
+import work_order_utility
+
+logger = logging.getLogger(__name__)
+
+def test_wo_inv_param(setup_config):
+    """ Testing work order request with invalid workload parameter value. """
+
+    # retrieve values from conftest session fixture
+    worker_obj = setup_config[0]
+    sig_obj = setup_config[1]
+    uri_client = setup_config[2]
+    private_key = setup_config[3]
+    err_cd = setup_config[4]
+
+    # input and output names
+    input_json_file = r'./work_order/workorder_input/wo_inv_param.json'
+    output_json_file_name = 'wo_inv_param'
+
+    # expected response
+    check_submit = {"error": {"code": 5}}
+    check_get_result = {"error": {"code": 2}}
+
+    # process work order
+    err_cd, response, worker_obj, sig_obj, encrypted_session_key = work_order_utility.process_work_order(input_json_file, output_json_file_name, worker_obj, sig_obj, uri_client, private_key, err_cd, check_submit, check_get_result)
```

Comment:
This line is too long and is hard to read.  Please limit lines to 80 characters.

Reply:
Fixed line length to below 80